internal/client/hub/hub.go

```diff
@@ -836,7 +836,7 @@ func ListExternalAPIs(filter string, pageSize int, pageToken string) (respBody [
 }
 
 func UpdateExternalAPI(externalApiID string, displayName string, description string,
-	endpoints []string, paths []string, externalUri string,
+	endpoints []string, paths []string, externalUri string, apiType string,
```

Comment:
shouldnt "apiType" arg be called "attributes"?

Reply:
No, this is correct. It is not attribute but `apiType`.